frontend/pages/admin/IntegrationsPage/cards/Sso/Sso.tsx

```diff
@@ -157,11 +169,34 @@ const Sso = ({
       },
     };
 
-    handleSubmit(formDataToSubmit);
+    if (await handleSubmit(formDataToSubmit)) {
+      setFormDirty(false);
+    }
   };
 
-  return (
-    <SettingsSection title="Single sign-on (SSO)">
+  const handleTabChange = useCallback(
+    (index: number) => {
+      if (
+        formDirty &&
+        // eslint-disable-next-line no-alert
+        !confirm("Switch tabs?\n\nChanges you made will not be saved.")
+      ) {
+        return;
+      }
+
+      setFormDirty(false);
```

Comment:
Note that this does _not_ reset values to what we got from app config. @noahtalerman / @rachaelshaw should we make a config GET call to reset?

Reply:
Per design review, that's definitely what we want to do, and I just pushed those changes.